### PR TITLE
docs: initialize zh docs

### PR DESCRIPTION
**Pull request by Timeless0911**
## Summary

initialize zh docs

## Checklist

- [ ] Tests updated (or not required).
- [x] Documentation updated (or not required).

### CODE REVIEW

File: website/docs/zh/index.mdx [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
 pageType: home
-titleSuffix: ' - 基于 Rsbuild 的库构建工具'
+titleSuffix: ' - 基于 Rsbuild 的 Library 开发工具'
```
Comment (fi3ework) [inline website/docs/zh/index.mdx:3]: Do we need to reserve `Library`, I think using  `库` is good enough.

Comment (fi3ework) [reply website/docs/zh/index.mdx:3]: cc @chenjiahan

File: website/docs/zh/guide/start/index.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# 介绍
+
+Rslib 是一个 Library 开发工具，它基于 [Rsbuild](https://rsbuild.dev/zh/) 精心设计的配置和插件，使库开发者得以复用 webpack 和 Rspack 繁荣的知识和生态系统。
```
Comment (Timeless0911) [inline website/docs/zh/guide/start/index.mdx:3]: `Library 开发工具` or `库开发工具`
`Library 开发者` or `库开发者`

File: website/docs/zh/index.mdx (hunk 1 above)
Comment (Timeless0911) [reply website/docs/zh/index.mdx:3]: Only called `Library 开发工具` in slogan, and in doc content, use `库` instead of `Library`?

Comment (chenjiahan) [reply website/docs/zh/index.mdx:3]: `库开发工具` LGTM

**Review by fi3ework — APPROVED** [review]

**Pull request merged**